README.md

```diff
@@ -1,5 +1,7 @@
 # Figma Plugins
 
-We currently have the following plugin:
+This library contains various plugins developed by the PX Blue team for use with [Figma](https://www.figma.com/).
 
--   [Variant Switcher](./variant-switcher/README.md): batch switch multiple component instances to variants sharing a same property.
+The following plugins are avalable:
+
+-   [Variant Switcher](./variant-switcher/README.md): recursively change all selected component instances from one variant / property to another.
```

Comment:
In Figma's jargon, 

"variant" is one... thing in a component. For example, `style=outlined, state=hovered, icon=no icon` is a variant in the component `button`. Think of a variant as a preset config of a button component.

"Property" is like the props. So in the example above, `style`, `state`, and `icon` would all be properties of the button component. 

I know it can be complicated to word this... is there a better way to phrase "from one variant / property to another"?

Reply:
@joebochill I'd like to get this and only this comment addressed so that I can publish the thing

Reply:
I'm confused...you said icon is a variant and then you said icon is a property.

You are probably more qualified to tweak the language to be correct than I am...I'm not as well-versed on Figma lingo

Reply:
"recursively changes component instances from one variant to another for a given property." I don't know.

Reply:
*recursively swaps component variants based on a specified property value.